fitnessllm_dataplatform/infrastructure/RedisConnect.py

```
@@ -0,0 +1,105 @@
+
+        Raises:
+            RedisError
+        """
+        self.open_connection()
+        try:
+            value = self.interface.get(key)
+
+            if value is None:
+                logger.info(f"Failed to get key '{key}'; does not exist")
+                return None
+            else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)